src/Image/Darkroom.php

```diff
@@ -93,6 +94,11 @@ protected function options(array $options = []): array
 			unset($options['bw']);
 		}
 
+		// normalize the sharpen option
+		if ($options['sharpen'] === true) {
+			$options['sharpen'] = 50;
```

Comment:
TBH I have no experience with the sharpen filter. Would you say that 50 is the most universal default?

Reply:
@bastianallgeier 50 is the default SimpleImage uses, see here: https://github.com/claviska/SimpleImage/blob/master/src/claviska/SimpleImage.php#L2059

The default value of 50 will be normalized to `0.5` with this line `$amount = max(1, min(100, $amount)) / 100;`. The resulting matrix the becomes (https://github.com/claviska/SimpleImage/blob/master/src/claviska/SimpleImage.php#L2064-L2067):

```
[-1, -1, -1],
[-1, 16, -1],
[-1, -1, -1],
```

Imagine, the Craft CMS image library, hardcodes the matrix with the same values: https://github.com/pixelandtonic/Imagine/blob/4d9bb596ff60504e37ccf9103c0bb705dba7fec6/src/Gd/Effects.php#L120

I guess the main question here would be if this value should be configurable or just simply a boolean.

Reply:
Sounds good to me!